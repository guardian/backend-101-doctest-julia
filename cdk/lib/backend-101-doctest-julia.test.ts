import { App } from 'aws-cdk-lib';
import { Template } from 'aws-cdk-lib/assertions';
import { Backend101DoctestJulia } from './backend-101-doctest-julia';

describe('The Backend101DoctestJulia stack', () => {
	it('matches the snapshot', () => {
		const app = new App();
		const stack = new Backend101DoctestJulia(app, 'Backend101DoctestJulia', {
			stack: 'deploy',
			stage: 'TEST',
		});
		const template = Template.fromStack(stack);
		expect(template.toJSON()).toMatchSnapshot();
	});
});
