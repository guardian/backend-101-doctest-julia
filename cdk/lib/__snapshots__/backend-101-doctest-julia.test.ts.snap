// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Backend101DoctestJulia stack matches the snapshot 1`] = `Object {}`;
